CactbotOverlay/CactbotOverlay.cs

```diff
@@ -247,6 +253,90 @@ public CactbotOverlay(CactbotOverlayConfig config)
       fast_update_timer_semaphore_.Release();
     }
 
+    public void EnableWatchers(bool check)
+    {
+      if (!init_ || this.watchers == null)
+      {
+        this.watcherState = check;
+        return;
+      }
+      // I doubt that the following code is actually necessary
+      if (this.watchers.Length > 0)
+        foreach (var watcher in this.watchers)
+          watcher.EnableRaisingEvents = check;
+    }
+ 
+    private void SetupFileWatcher(string url)
+    {
+      if (!init_)
```

Comment:
init_ will always be true when you get here, just based on where this is called.